index.html

```diff
@@ -53,5 +53,158 @@ <h2 class="job">I'm a software developer</h2>
     </section>
+    <section class="works">
+      <h2 class="title">My Recent Works</h2>
+      <hr />
+      <div class="cards">
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li><a href="#">Ruby on rails</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">html</a></li>
+            </ul>
+            <a class="see_project" href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li><a href="#">Ruby on rails</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">html</a></li>
+            </ul>
+            <a class="see_project" href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li><a href="#">Ruby on rails</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">html</a></li>
+            </ul>
+            <a class="see_project" href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li><a href="#">Ruby on rails</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">html</a></li>
+            </ul>
+            <a class="see_project" href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
```

Comment:
Nice Job. You made a good job so far. Kindly you can add 'URL to the original websites of social medias.